{{cookiecutter.project_name}}/mkdocs.yml

```diff
@@ -10,6 +10,8 @@ theme:
 plugins:
   - search
   - mermaid2
+  - mermaid2:
+      version: 9.3.0
```

Comment:
The latest release of _mermaid.js_ is 10.9.1: https://github.com/mermaid-js/mermaid/releases

I think it's okay for us to specify a stable v10 release, like 10.9.0 or something.